src/plugins/drawing-tool/components/selection-drawing-tool.tsx

```diff
@@ -21,7 +21,7 @@ export class SelectionDrawingTool extends DrawingTool {
     };
     const handleMouseUp = (e2: MouseEvent) => {
       e2.preventDefault();
-      drawingLayerView.endSelectionBox(addToSelectedObjects);
+      drawingLayerView.endSelectionBox(addToSelectedObjects, e2);
```

Comment:
If `e.shiftKey` was added to the value of  `addToSelectedObjects` above then I think you don't need to pass e2 here.